src/Cli/dotnet/commands/dotnet-test/TestingPlatformOptions.cs

```diff
@@ -57,5 +57,11 @@ internal static class TestingPlatformOptions
             Description = LocalizableStrings.CmdProjectDescription,
             Arity = ArgumentArity.ExactlyOne
         };
+
+        public static readonly CliOption<string> BinLogOption = new("-bl")
```

Comment:
There are multiple syntaxes to the binlog, is this code enough to support them all? See https://learn.microsoft.com/visualstudio/msbuild/msbuild-command-line-reference?view=vs-2022#switches-for-loggers

Reply:
Consider reusing the logic we have in `dotnet run` here: https://github.com/dotnet/sdk/blob/a9ebd73da1c3d4a1913f085cf9e9148ef670472c/src/Cli/dotnet/commands/dotnet-run/RunCommand.cs#L338-L351

We have similar problems - both of our tools should reuse common functionality.

Reply:
It might be even better to have a helper to reuse code rather than copy pasting so that we don't get out of date.

Reply:
Totally agree - I'd say that now that you have the same problem it should be extracted.